www/src/components/docs-breadcrumb.js

```diff
@@ -0,0 +1,91 @@
+import React from "react"
+import { Link } from "gatsby"
+import ChevronRight from "react-icons/lib/md/chevron-right"
+import ChevronLeft from "react-icons/lib/md/chevron-left"
+import getActiveItem from "../utils/sidebar/get-active-item"
+import getActiveItemParents from "../utils/sidebar/get-active-item-parents"
+import { mediaQueries, space, fontSizes } from "../utils/presets"
+
+const Separator = ({ character = <ChevronRight /> }) => (
+  <span style={{ margin: `0px ${space[1]}` }} role="presentation">
+    {character}
+  </span>
+)
+
+const BreadcrumbNav = ({ children, mobile = false }) => (
+  <nav
+    aria-label="breadcrumb"
+    css={{
+      fontSize: fontSizes[1],
+      display: `${mobile ? `inherit` : `none`}`,
+      [mediaQueries.md]: {
+        display: `${mobile ? `none` : `inherit`}`,
+      },
+      marginBottom: space[2],
+    }}
+  >
+    {children}
+  </nav>
+)
+
+const Breadcrumb = ({ itemList, location }) => {
+  const activeItem = getActiveItem(itemList.items, location, undefined)
+  const activeItemParents = getActiveItemParents(itemList.items, activeItem, [])
+
+  // return a shorter version of the breadcrumb on the intro page
+  // because the docs intro page isn't generated from markdown
+  if (activeItem.title === `Introduction`) {
+    return (
+      <>
+        {/* only the breadcrumb nav of the proper viewport is displayed */}
+        <BreadcrumbNav>
+          <Link to="/">Home</Link>
+          <Separator />
+          Docs
```

Comment:
This renders on the Tutorials and Contributing sections, but still says `docs`. We should fix that.

Reply:
Just fixed this, now it will take the top level of the itemsList passed in and use it for the link and title displayed there which should reflect the highest item in sidebar